application/console/components/AwsStartupAction.php

```diff
@@ -38,16 +41,24 @@ public function performAction()
             ];
             $this->createProject($projectName, $cache, $source, $logger);
 
-           // Build publish role if necessary
+            // Build publish role if necessary
+            // Copy default file
             $projectName = 'publish_app';
+            $prefix = 'arn:aws:s3:::';
+            $bucket = S3::getArtifactsBucket();
+            $baseUrl = 'default/default.zip';
+            $defaultLocation = $prefix . $bucket . "/" . $baseUrl;
+            $defaultLocation = str_replace('"', "", $defaultLocation);
```

Comment:
Above you did this style of str_replace on just the `S3::getArtifactsBucket()` and then here again against the whole path.  Is the problem that sometimes the result of `S3::getArtifactsBucket()` includes quotes in the value.  Will this cause problems in other situations?  Should it be fixed in `S3::getArtifactsBucket()` and possibly other S3 methods?